### PR TITLE
feat: update for libvips v8.14.5 and bindgen 0.69.0

### PR DESCRIPTION
**Pull request by negezor**
The libvips library has moved to the new meson + ninja builder. I couldn't wait for crate to update myself, so I thought I'd suggest a PR. Let me know if there are any issues with this PR here.

### CODE REVIEW

**Comment by yaa110** [comment]
@augustocdias could you please review and merge it?

**Comment by negezor** [comment]
I also upgraded num-derive to 0.4 to go to syn 2. Now the minimum MSVR is 1.56.

**Comment by augustocdias** [comment]
looks good to me but I'm not part of the organisation anymore. I can't approve/merge PRs.

I don't know if you want to tackle this, but a mistake I did was pushing the generated bidings from bindgen to github instead of always generating on build time. This will cause issues on different platforms as types has different types in each. If you're running on docker or linux (and probably mac as well) this should be fine, but on windows this will certainly fail.

**Comment by yaa110** [comment]
@negezor would you mind creating a new repo for libvips and publishing it to crates.io (e.g. as vips)?

**Comment by GamerBene19** [comment]
@augustocdias Do you still have contact to someone from the olxgroup-oss organization? It would be nice if this repository could still be maintained (by the community). Perhaps someone could be given the necessary access rights to this repository.

Otherwise I think a fork (and republishing) is necessary. It looks like [vips](https://crates.io/crates/vips) name is already taken so one would have to find a new one

File: generator/Dockerfile [hunk 1]
```diff
@@ -1,13 +1,15 @@
 # (c) Copyright 2019-2023 OLX
-FROM rust:1.66.0-alpine3.17
+FROM rust:1.73.0-alpine3.18
 
-ENV VIPS_VERSION=8.13.3
+ENV VIPS_VERSION=8.14.5
 
 RUN apk add --update --no-cache --repository=https://dl-cdn.alpinelinux.org/alpine/v3.17/main \
```
Comment (bassco) [inline generator/Dockerfile:6]: @negezor - thanks for the work you have done with the PR.

> RUN apk add --update --no-cache --repository=https://dl-cdn.alpinelinux.org/alpine/v3.17/main \

Can you bump this to *3.18* to match the `FROM`

**Review by bassco — CHANGES_REQUESTED** [review]
Once again thanks for the PR. Looks good. Just the small item to take care of and then it can be merged and published.

Apologies for the slow response.

**Comment by negezor** [comment]
@bassco Look like that's it. Perhaps I'll come back later, as libvips version 8.15.0 is just around the corner 😄

**Comment by bassco** [comment]
@negezor - I'll publish the crate in the next few hours once I get onto the work machine

cc @BogdanVidrean

**Pull request merged**